src/scripts/match-mondo-sources-all-lexical.py

```diff
@@ -118,9 +119,9 @@ def run(input: str, config: str, rules: str, rejects: str, output: str):
     save_lexical_index(lexical_index, OUT_INDEX_DB)
 
     if rules:
-        msdf = lexical_index_to_sssom(oi, lexical_index, ruleset=ruleset, meta=meta)
+        msdf = lexical_index_to_sssom(oi, lexical_index, ruleset=ruleset, prefix_map=converter)
```

Comment:
## Redundancy with #397
The main difference is that #397 does passes `meta=None` to `lexical_index_to_sssom()`, whereas #394 passes a `Converter`. Supposedly #397 approach is fine.

Everything else in that #394 is now cosmetic, so i can merge just those changes afterwards or ignore and close.